python-shaptools.spec

```diff
@@ -52,6 +52,17 @@ BuildArch:      noarch
 %description
 API to expose SAP HANA functionalities
 
+%package -n python3-shaptools-venv-salt-minion
```

Comment:
I see `python3` used always. What if we need the py2 version?
Or is it irrelevant?

Reply:
Hmm, the "venv-salt-minion" package is only Python 3.

Are you supporting shap based formulas on Python 2 OSes?

Reply:
We were when I implemented this package 5 years ago. Right now, I'm not really sure to be honest

Reply:
I would also simply not support the py2 version use case in the "venv-salt-minion" use case.
Maybe a simple `%if %{without python2}` condition could be wrapped around the sub-package?

Reply:
Thanks for the notes :+1: 

So Salt Bundle is only Python 3, and from what I can see, formulas like "saphanabootstrap" are installing `python3-shaptools` depending on the `pythonversion` that is processing the Salt formula: https://github.com/SUSE/saphanabootstrap-formula/blob/main/hana/packages.sls#L32

Therefore if Salt Bundle is the one processing the formula, which is always Python 3, then the "python3-shaptools" package will be installed and, due to this PR, the new `python3-shaptools-venv-salt-minion` package will get automatically installed. We are all fine here.

It doesn't really make to have a python 2 flavor of this integration with the Salt Bundle.

> I would also simply not support the py2 version use case in the "venv-salt-minion" use case. Maybe a simple `%if %{without python2}` condition could be wrapped around the sub-package?

Agree as mentioned. It doesn't make sense a Python 2 flavor for the "venv-salt-minion".

IIUC the `%if %{without python2}` is not really needed here - we always want to build the `python3-shaptools-venv-salt-minion` package, AFAICS the "python-shaptools" always build python3 subpackages and for some OSes it additionally builds the python2 flavor.